ls1intum/Artemis
# [Enhancement] - Improve course detail statistics

### Checklist

### Motivation and Context


We currently show the active users in the last 4 weeks, which leaves a lot of space.

### Description

We now display more weeks at once to provide more information in once glance. We display 16 weeks at once.

### Steps for Testing

**Ideally before deploying the PR, look at some Active user charts in the detail page of 1 or two courses and screenshot or just keep in mind those numbers in the last 16 weeks**

1. Log in to Artemis
2. Navigate to Course Administration
3. Click on the header of an Course and make to navigate into the course detail page
4. Make sure no errors are thrown
5. Compare the values of the active users line chart with the values before deploying the PR if they are the same


### Screenshots


Before:


<img width="786" alt="Screenshot 2021-07-24 at 10 39 16" src="https://user-images.githubusercontent.com/44401560/126862992-9e238198-6f25-4b39-b71f-5af3924d54d0.png">
<img width="785" alt="Screenshot 2021-07-24 at 10 39 26" src="https://user-images.githubusercontent.com/44401560/126862993-9e9bbd38-aea1-4370-ad95-30944cfbd0ca.png">


Now:
<img width="784" alt="Screenshot 2021-07-24 at 10 42 21" src="https://user-images.githubusercontent.com/44401560/126862995-349942af-2d12-4ee8-8fc5-7f306f59b003.png">
